src/joystick.cc

```diff
@@ -0,0 +1,121 @@
+#include "joystick.h"
```

Comment:
For C++ part, we should follow Google C++ Style Guide and [Node.js habits](https://github.com/nodejs/node/blob/master/doc/guides/cpp-style-guide.md):

1. `snake_case` for local variables and parameters;
2. `_snake_case` for private class fields;
3. `snake_case` for C-like structs;
4. `CamelCase` for methods, functions, and classes;
5. Being explicit about types is usually preferred over using `auto` except things like `xxx::iterator`, etc.

We can refer to https://github.com/nodejs/node/blob/master/doc/guides/cpp-style-guide.md to get more rules.

Reply:
Ok no problem :)
Thanks for the hint! (Fixed I hope).